### PR TITLE
feature(ui-ux): update UI for address selection bottom sheet

### LINKED ISSUE #2382: `address` - updated address switcher screen
#### What would you like to be added:
Linked - https://github.com/DeFiCh/wallet/issues/2281

- [ ] Remove the Address screen, use bottom sheet instead
- [ ] Will display the avatar of the selected wallet
- [ ] Receive in Wallet
- [ ] View in DeFiScan
- [ ] Display number of wallets
- [ ] Clicking on a row, will select the address
image

### PR DESCRIPTION
**Pull request by kyleleow**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
- To update UI for address selection bottom sheet to include full copy-able address
- To prepare for steps to add/edit address label
- To replace address control screen with bottom sheet

#### Which issue(s) does this PR fixes?:

Fixes #2382

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [x] Unit tests*
- [x] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

**Review by thedoublejay — CHANGES_REQUESTED** [review]
On dark mode (android), there's a border above the close button

File: mobile-app/app/screens/AppNavigator/screens/Balances/components/BottomSheetAddressDetail.tsx [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+  }, [blockCount])
+
+  const CreateAddress = useCallback(() => {
+    if (!canCreateAddress) {
+      return <></>
+    }
+
+    return (
+      <ThemedTouchableOpacity
+        light={tailwind('bg-white border-gray-100')}
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Balances/components/BottomSheetAddressDetail.tsx:106]: This one is not that visible also on Dark mode. Should we consider changing? @jtlntno-des

File: mobile-app/app/screens/AppNavigator/screens/Balances/screens/SendScreen.tsx [hunk 2]
```diff
@@ -128,8 +127,8 @@ export function SendScreen ({
     setHasBalance(totalBalance.isGreaterThan(0))
   }, [JSON.stringify(tokens)])
 
-  useEffect(() => {
-    setBottomSheetScreen([
+  const bottomSheetScreen = useMemo(() => {
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Balances/screens/SendScreen.tsx:130]: Let's remove `useMemo` here, might be a potential cause of the disappearing bottom sheet

**Review by thedoublejay — CHANGES_REQUESTED** [review]
Screenshot_20220323-090505_Expo Go

Address is cut-off

**Review by thedoublejay — CHANGES_REQUESTED** [review]
Another UI issue if you can add, on bigger numbers, the notif is too close to the address

Screenshot_20220323-091925_Expo Go

File: mobile-app/app/screens/AppNavigator/screens/Balances/screens/SendScreen.tsx (hunk 2 above)
Comment (kyleleow) [reply mobile-app/app/screens/AppNavigator/screens/Balances/screens/SendScreen.tsx:130]: adding `useMemo()` is actually to fix disappearing bottom sheet

File: mobile-app/app/screens/AppNavigator/screens/Balances/components/BottomSheetAddressDetail.tsx (hunk 1 above)
Comment (thedoublejay) [reply mobile-app/app/screens/AppNavigator/screens/Balances/components/BottomSheetAddressDetail.tsx:106]: Will be tackled on a different issue

**Comment by kyleleow** [comment]
> Another UI issue if you can add, on bigger numbers, the notif is too close to the address
> 
The spacing actually looks fine on my android, not sure why it is different between different android devices. 
image

@thedoublejay If i add more margin here it would become slightly bigger space in my android, but it might fixed your issue, should i proceed?
E.g.:
image

**Pull request merged**